docs/tools/index.md

```diff
@@ -0,0 +1,71 @@
+# Development tools
+
+## CiviCRM-specific tools
```

Comment:
`1.` In my gut, I like having the list of tools in `civicrm-buildkit.git/README.md`. It conveys the concept/essence of the repo.

`2.` The way you've organized this does suggest a somewhat different perspective from mine. Compare these framings:
 * Buildkit is a bunch of Civi tools, and it also incorporates some external tools as-needed. (Oddly, some of the internal tools -- like `cv`, `civix`, `civistrings`, `civicrm-upgrade-test` -- are actually separate projects.)
 * Developers in the Civi ecosystem need 20 some tools. Tools have many different origins/purposes (PHP dev, JS dev, Civi dev, Drupal dev, WP dev, etc.) Buildkit is a downloader which fetches all of them. (Oddly, some tools -- like `civibuild` and `cividist` -- are developed in `civicrm-buildkit.git`.)

For me, `buildkit` is mostly a package or downloader. I'd be happier if we could move `civibuild` and `cividist` into separate projects. (But status-quo arose because the cost-benefit of making that move isn't particularly great.)